packages/datadog-plugin-aws-sdk/src/services/bedrockruntime.js

```diff
@@ -0,0 +1,207 @@
+'use strict'
+
+const BaseAwsSdkPlugin = require('../base')
+const log = require('../../../dd-trace/src/log')
+
+const _AI21 = 'ai21'
+const _AMAZON = 'amazon'
+const _ANTHROPIC = 'anthropic'
+const _COHERE = 'cohere'
+const _META = 'meta'
+const _STABILITY = 'stability'
+
+class BedrockRuntime extends BaseAwsSdkPlugin {
+  static get id () { return 'bedrock runtime' }
+
+  generateTags (params, operation, response) {
+    let tags = {}
+    let modelName = ''
+    let modelProvider = ''
+    const modelMeta = params.modelId.split('.')
+    if (modelMeta.length === 2) {
+      [modelProvider, modelName] = modelMeta
+      modelProvider = modelProvider.toUpperCase()
+    } else {
+      [, modelProvider, modelName] = modelMeta
+      modelProvider = modelProvider.toUpperCase()
+    }
+
+    const shouldSetChoiceIds = modelProvider === 'COHERE' && !modelName.includes('embed')
```

Comment:
```suggestion
    const shouldSetChoiceIds = modelProvider === 'cohere' && !modelName.includes('embed')
```
if we change the modelProvider to just be normal case (or even force it to lowercase)

Reply:
I forced everything now to uppercase and replaced that with the constant for cohere